packages/block-library/src/button/edit.js

```diff
@@ -221,7 +221,7 @@ function ButtonEdit( props ) {
 					anchorRef={ ref?.current }
 					focusOnMount={ isEditingURL ? 'firstElement' : false }
 					__unstableSlotName={ '__unstable-block-tools-after' }
-					__unstableShift
+					shift
```

Comment:
Are we intentionally keeping the legacy prop here:

https://github.com/WordPress/gutenberg/blob/5662664ea1dbd8f422fdce77516cad33b5b83422/packages/block-library/src/navigation-link/edit.js#L848

Reply:
Nope, just a miss in my find-and-replace 🙈 

Done in https://github.com/WordPress/gutenberg/pull/43845/commits/a8421c519795ef2b496ac9f2387f56a58e8d5148